lib/open_pantry/web/models/stock_distribution.ex

```diff
@@ -44,13 +45,14 @@ defmodule OpenPantry.StockDistribution do
     end
   end
 
-  defp per_person_ok?(stock_distribution = %StockDistribution{ stock: %Stock{ max_per_person: nil } }), do: true
-  defp per_package_ok?(stock_distribution = %StockDistribution{ stock: %Stock{ max_per_package: nil } }), do: true
+  defp per_person_ok?(%StockDistribution{ stock: %Stock{ max_per_person: nil } }), do: true
```

Comment:
Minor cleanup, convention is no space between multiple heads of the same function, yes between different functions... so good to put these adjacent too each other (and bad on me for splitting up) but can also make consistent line breaks between them per convention